ext-plugin.fbs

```diff
@@ -86,6 +86,8 @@ table Req {
     path:string;
     args:[TextEntry];
     headers:[TextEntry];
+    // To change the upstream response header
+    resp_headers:[TextEntry];
```

Comment:
We don't need to add the response header to the req we send to the runner? The RPC happens in an early stage, before we get real upstream response header.

Reply:
Ok, I'll correct it right away.